<?xml version="1.0"?>
<launch>
    <node name="controller" pkg="pid" type="controller" ns="x" output="screen" >
      <param name="Kp" value="-5.0" />
      <param name="Ki" value="-0.1" />
      <param name="Kd" value="0.0" />
      <param name="upper_limit" value="10" />
      <param name="lower_limit" value="-10" />
      <param name="windup_limit" value="10" />
      <param name="max_loop_frequency" value="100.0" />
      <param name="min_loop_frequency" value="20.0" />
      <remap from="setpoint" to="/ref_body_x" />
     </node>



    <node name="controller" pkg="pid" type="controller" ns="y" output="screen" >
      <param name="Kp" value="-4.0" />
      <param name="Ki" value="-0.1" />
      <param name="Kd" value="0.0" />
      <param name="upper_limit" value="10" />
      <param name="lower_limit" value="-10" />
      <param name="windup_limit" value="10" />
      <param name="max_loop_frequency" value="100.0" />
      <param name="min_loop_frequency" value="20.0" />
      <remap from="setpoint" to="/ref_body_y" />
     </node>


    <node name="controller" pkg="pid" type="controller" ns="z" output="screen" >
      <param name="Kp" value="-4.0" />
      <param name="Ki" value="-0.1" />
      <param name="Kd" value="0.0" />
      <param name="upper_limit" value="10" />
      <param name="lower_limit" value="-10" />
      <param name="windup_limit" value="10" />
      <param name="max_loop_frequency" value="100.0" />
      <param name="min_loop_frequency" value="20.0" />
      <remap from="setpoint" to="/ref_body_z" />
     </node>

    <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" />

</launch>

